### ISSUE TITLE
Duplicate values in nested object values resulting from @cypher field

### ISSUE THREAD

**Issue by johnymontana** [neo4j-graphql/neo4j-graphql-js#11]
Originally reported in this [Launchpad](https://launchpad.graphql.com/zpjz95k47)

The query

~~~
{
  Movie(title: "Kill Bill: Vol. 1") {
    title
    genres {
      name
    }
    similar(first:3) {
      title
      genres {
        name
      }
    }
  }
}
~~~

results in this Cypher query

~~~
MATCH (movie:Movie {title:"Kill Bill: Vol. 1"}) 
RETURN movie { .title ,
	genres: [(movie)-[:IN_GENRE]->(movie_genres:Genre) | movie_genres { .name }] ,
	similar: [ x IN apoc.cypher.runFirstColumn("WITH {this} AS this MATCH (this)--(:Genre)--(o:Movie) RETURN o LIMIT 5", {this: movie}, false) | x 
  		{ 
    		.title ,
			genres: [(movie_similar)-[:IN_GENRE]->(movie_similar_genres:Genre) | movie_similar_genres { .name }] }][..3] 
    	} AS movie SKIP 0
~~~

and returns duplicate genre names:

~~~
{
"data": {
    "Movie": [
      {
        "title": "Kill Bill: Vol. 1",
        "genres": [
          {
            "name": "Thriller"
          },
          {
            "name": "Crime"
          },
          {
            "name": "Action"
          }
        ],
        "similar": [
          {
            "title": "Usual Suspects, The",
            "genres": [
              {
                "name": "Adventure"
              },
              {
                "name": "Adventure"
              },
              {
                "name": "Adventure"
              },
              {
                "name": "Adventure"
              },
              {
                "name": "Adventure"
              },
              {
                "name": "Adventure"
              },
              {
                "name": "Adventure"
              },
              {
                "name": "Adventure"
              },
              {
                "name": "Adventure"
              },
...
~~~

**Comment by judas-christ** [comment]
This looks very much like #30. Is this still a problem or is it fixed since #31?

**Comment by johnymontana** [comment]
Yup, good catch @judas-christ. This was fixed by your PR #31. Thanks!
